backend/index.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let sessionCounter = 0;
+let counter;
+
+try {
+  fs.existsSync("temp/my-counter-visited-times.txt")
+    ? (counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
+    : (counter = 0);
+} catch (e) {
+  console.log(e);
+}
+
+app.get("/", (req, res) => {
+  sessionCounter++;
+  counter = parseInt(counter);
```

Comment:
🗡️ It should always be a number during the assignment in other places.

Reply:
Also here you can do `counter++` when you change it to be a number and later instead of `sessionCounter + counter` just use `counter`